ci/documentation_updater/README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation updater
+
+This script updates description in `model.yml` file according to description in markdown documentation from section *Use Case and High-Level Description*.
+
+## Prerequisites
+
+Install `ruamel.yaml` package:
+```
+pip install ruamel.yaml
+```
+
+## Usage
+
+To update description of single model:
+```
+python documentation_updater.py -d <OMZ dir>/models/public/<model dir>
+```
+
+To check descriptions of all public models:
+```
+python documentation_updater.py -d <OMZ dir>/models --mode check
+```
+OR
+```
+python documentation_updater.py -d <OMZ dir>/models --mode check
+```
```

Comment:
These two alternatives seem to be the same. Also, both the `-d` and `--mode` options are set to their default values and can be omitted.

Reply:
Well, you deduplicated them, but what about my second suggestion?

Reply:
Yes, thanks, I mixed default values up